enrolment-service/Dockerfile

```diff
@@ -1,16 +1,17 @@
-FROM alpine
+FROM alpine:3.9
```

Comment:
Had to lock it down to an older version to keep compatibility with the vastly outdated Django version, else it wouldn't even run with the code existing in develop (a heap of deprecated requirements).

This Django deal ought to be addressed somewhat soon too.